src/types/api.ts

```diff
@@ -158,6 +158,12 @@ export type UserInfo = {
   display_name: string | null;
 };
 
+export type Mention = UserInfo & {
+  userid: string;
+  username: string;
+  link: string;
```

Comment:
This may not be entirely correct. For example, during my testing I was getting `null` links. We'll have to clarify this with the BE.